src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Providers/Providers/AzureFilesPsBackupProvider.cs

```diff
@@ -553,7 +554,51 @@ public List<ItemBase> ListProtectedItems()
 
         public ResourceBackupStatus CheckBackupStatus()
```

Comment:
Please use the CheckBackupStatus API instead of doing a brute-force search through all the vaults and items. Check with the SDK team for the exact signature.